components/EventsComponent.vue

```diff
@@ -18,17 +18,31 @@ const d = Object.fromEntries(Object.entries(data).filter(([k, v]) =>
 ).sort())
 </script>
 
+<style scoped>
+.eventsList h3:first-of-type {
+  border-top-width: 0;
+  padding-top: 0;
+}
+.eventsList h3 {
+  border-top: 1px solid var(--vp-c-divider);
+  margin-top: 18px;
+  padding-top: 18px;
+}
+.eventsList :deep(table p) {
+  margin: 0;
+}
+</style>
+
 <template>
- <template v-for="(v, k) in d">
-  <h3 :id="k" tabindex="-1">
-   <code>{{ k }}</code>
-   <a class="header-anchor" :href="'#' + k"></a>
-  </h3>
+ <span class="eventsList">
```

Comment:
Do not use `<span>` tags for non-inline items. Here it would make sense to use a `<section>`.

Reply:
Changed.